BugDoom - web app/src/main/java/com/infoshareacademy/pl/controller/TrackController.java

```diff
@@ -1,24 +1,73 @@
 package com.infoshareacademy.pl.controller;
 
+import com.infoshareacademy.pl.model.Location;
 import com.infoshareacademy.pl.model.Track;
 import com.infoshareacademy.pl.repository.TrackRepository;
-import com.infoshareacademy.pl.service.TrackService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
 
+import javax.validation.Valid;
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 
-@RestController
-public class TrackController {
 
-    private final TrackService trackService;
+@Controller
+public class TrackController {
```

Comment:
W całej klasie rozjechało Ci się formatowanie, 
masz zbędne importy
W dodatku brakuje konsekwencji w adresach url tzn jak wyświetlasz formularz pod adresem  @GetMapping("/tracks/create") to wysłanie powinno być pod @PostMapping("/tracks/create") a nie @PostMapping("/tracks")